package.json

```diff
@@ -6,7 +6,10 @@
     "app-config",
     "app-config-webpack-plugin",
     "app-config-inject",
-    "examples/*",
+    "app-config-react-native-transformer",
+    "examples/cli-project",
+    "examples/frontend-webpack-project",
+    "examples/nodejs-server",
```

Comment:
FYI. As we know, react native doesn't play nicely with mono-repos. I tried an 'exclusion' pattern here, but didn't seem to work. ie:
```json
[
    "examples/*",
    "!examples/react-native-project"
]
```

Reply:
This is basically what `nohoist` is for. It's a bitch though.

Reply:
Yeah I started looking into `nohoist`, but didn't think it was worth the rabbit hole & slow down the release of the RN transformer.